test/ParserTests.hs

```diff
@@ -76,8 +76,8 @@ parseLine' pars = parseAll pars ""
 testParseBinaryOp :: Test
 testParseBinaryOp = TestCase (
   assertEqual "Check Parsing of Binary Op"
-  True
-  (parseLine' expr "40 + 2" == Right (Binop Plus (I 40) (I 2))))
+  Right (Binop Plus (I 40) (I 2)))
```

Comment:
This is failing in the CI

Reply:
this fails because parses now have a bunch of annotations pasted on.